types/src/chainspec/vm_config/host_function_costs.rs

```diff
@@ -301,6 +316,125 @@ pub struct HostFunctionCosts {
     pub emit_message: HostFunction<[Cost; 4]>,
 }
 
+impl Add for HostFunctionCosts {
+    type Output = HostFunctionCosts;
+
+    fn add(self, rhs: Self) -> Self::Output {
+        Self {
+            read_value: self.read_value + rhs.read_value,
```

Comment:
This could use some macro-magic to avoid copy&paste mistakes, or rework of `HostFunctionCosts`.
Not in this PR, though.

Reply:
I went with the default aversion to using macros here 😁 but it does seem that we use `derive_more` in some other places in the project so I can switch to that for `Add`.

Reply:
Changed here: https://github.com/casper-network/casper-node/pull/4344/commits/ffe7e18b9a7139defaa1cebf8c97bde61722a229